src/options/smt_options.toml

```diff
@@ -620,25 +620,40 @@ header = "options/smt_options.h"
 [[option]]
   name       = "solveBVAsInt"
   category   = "undocumented"
-  long       = "solve-bv-as-int=N"
-  type       = "uint32_t"
-  default    = "0"
+  long       = "solve-bv-as-int"
+  type       = "bool"
+  default    = "false"
   read_only  = true
   help       = "attempt to solve BV formulas by translation to integer arithmetic (experimental)"
 
 [[option]]
-  name       = "bvToIntIand"
+  name       = "solveBVAsIntMode"
   category   = "undocumented"
-  long       = "bv-to-int-iand"
-  type       = "bool"
-  default    = "false"
+  long       = "solve-bv-as-int-mode=MODE"
+  type       = "SolveBVAsIntMode"
+  default    = "SUM"
+  help       = "mode for translating BVAnd to integer"
+  help_mode  = "solve-bv-as-int modes."
+[[option.mode.SUM]]
+  name = "sum"
+  help = "Generate a sum expression for each bvand instance, based on the value in --solbv-bv-as-int-granularity"
+[[option.mode.IAND]]
+  name = "iand"
+  help = "Translate bvand to the iand operator, ignoring --solve-bv-as-int-granularity. Granularity for iand is set separately by --iand-granularity option (experimental)"
```

Comment:
Why this distinction? Couldnt we reuse the same option (since it looks like they are used in a mutually exclusively way)?

Reply:
sure i can do that. How would we call it? :) Think of a user who wants to play with iand directly and has no idea what the preprocessing pass is, and on another user who wantes the preprocessing pass but doesn't know iand. Maybe... --?integer-bvand-granularity?

Reply:
Addressed in recent commit.

Reply:
Yeah, that sounds good :)